packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -154,3 +156,92 @@ ErrorReporter showErrorSnackBar(BuildContext context) {
     );
   };
 }
+
+Widget clearButton({
+  Key key,
+  double minIncludeTextWidth,
+  @required VoidFunction onPressed,
+}) {
+  return OutlineButton(
+    key: key,
+    onPressed: onPressed,
+    child: MaterialIconLabel(
+      Icons.block,
+      'Clear',
+      minIncludeTextWidth: minIncludeTextWidth,
+    ),
+  );
+}
+
+List<Widget> recordStopButtons({
```

Comment:
Suggestion: 
Build the containing widget that presents these two buttons, or separate this list into two widgets: one for record and one for stop.

Reply:
separated buttons and added documentation